scipy/stats/_distr_params.py

```diff
@@ -125,6 +125,7 @@
 distdiscrete = [
     ['bernoulli',(0.3,)],
     ['betabinom', (5, 2.3, 0.63)],
+    ['betanbinom', (5, 4.3, 1)],
```

Comment:
I recommend changing the default parameters. Hopefully the default kurtosis test passes in that case. With a=4.3, the kurtosis estimate is well-defined but its estimate is not stable, I assume that also explains why expect has problems. Maybe take (5, 9.3, 1)